rpgs/templates/rpgs/index.html

```diff
@@ -49,7 +49,7 @@ <h4 class="card-text">Show...</h4>
 {% block body %}
     {% if request.user.is_authenticated %}
         <p><a href="{% url 'rpgs:create' %}">
-            <button class="btn btn-block btn-outline-primary">Create event</button>
+            <button class="btn btn-block btn-success">Create event</button>
```

Comment:
Are you sure you didn't mean btn-outline-success. Nowhere else in the website uses the filled style (though if you tested in darkmode you might have missed this as they look the same)

Reply:
I tested on lightmode, and only did this on agreement from Ares - while nowhere else on the website uses this style, we wanted to make the button look as obvious as humanly possible given the sheer number of people who are surprised that they can create events on the website.